Contrib/NP_Score/npscorer.py

```diff
@@ -21,56 +21,70 @@
 
 
 def readNPModel(filename=os.path.join(os.path.dirname(__file__), 'publicnp.model.gz')):
-  sys.stderr.write("reading NP model ...\n")
-  fscore = pickle.load(gzip.open(filename))
-  sys.stderr.write("model in\n")
-  return fscore
+    sys.stderr.write("reading NP model ...\n")
+    fscore = pickle.load(gzip.open(filename))
+    sys.stderr.write("model in\n")
+    return fscore
 
 
-def scoreMol(mol, fscore):
-  if mol is None:
-    raise ValueError('invalid molecule')
-  fp = rdMolDescriptors.GetMorganFingerprint(mol, 2)
-  bits = fp.GetNonzeroElements()
+def scoreMolWConfidence(mol, fscore):
+    """Next to the NP Likeness Score, this function outputs a confidence value
+    between 0..1 that descibes how many fragments of the tested molecule
+    were found in the model data set (1: all fragments were found).
+
+    Returns tuple(NP Likeness, Confidnce)"""
```

Comment:
Use a `namedtuple` to return annotated results. 

Typo: Confidnce => Confidence

Reply:
Do you want to comment on the suggestion to use `namedtuple`? It still behaves like a tuple, but also allows access to individual fields by name.
```
NPLikeness = namedtuple('NPLikeness', 'npLikeness,confidence')

return NPLikeness(likeness, confidence)
```
and then you can use it like this:
```
score = scoreMolWConfidence(mol, fscore)
score.npLikeness == score[0]
score.confidence == score[1]
```

Reply:
Personally I don't like Namedtuples too much, but it is probably a good suggestion. Will add a PR tomorrow.